packages/components/atoms/f-image-tile/src/components/ImageTile.vue

```diff
@@ -24,11 +24,13 @@
             class="is-visuallyHidden"
             :class="$style['c-imageTile-checkbox']"
             data-test-id="image-tile-input"
+            :tabindex="!isLink ? 0 : false"
```

Comment:
I thought tabindex can only have integers as values?

Reply:
Yes that's correct 👍 
With the above if `isLink` is true the `tabindex` attribute won't be output.